doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/10_V6_to_V10.md

```diff
@@ -1,12 +1,57 @@
-# Upgrading Pimcore from Version 6.9 to Version 10
+# Upgrading Pimcore from Version 6.x to Version 10
 
-# Flush your `/tmp` directories
+## System Requirements
+ - PHP >= 8.0
+ - Apache >= 2.4
```

Comment:
Is `Apache` really required? We use `nginx`, for example ;)

Reply:
well, this is about what has changed on Pimcore X 😉 changed headings e.g. `System Requirement changes`